packages/calcite-components/src/components/functional/Validation.tsx

```diff
@@ -15,14 +17,18 @@ export const CSS = {
 };
 
 export const Validation: FunctionalComponent<ValidationProps> = ({
+  ariaLive,
   scale,
   status,
+  id,
   icon,
   message,
 }) => (
   <div class={CSS.validationContainer}>
     <calcite-input-message icon={icon} scale={scale} status={status}>
-      {message}
+      <span aria-live={ariaLive ? "polite" : "off"} id={id}>
```

Comment:
nitpick: do we need to create a span element here or could the aria-live and id be put on the `calcite-input-message`?